appveyor.yml

```diff
@@ -1,10 +1,15 @@
-os: Visual Studio 2015
+os: Visual Studio 2017
```

Comment:
I wonder if it would be better to go to 2019 🤔

Reply:
Should work as well. Trying that now